gui/options.cpp

```diff
@@ -1922,18 +1922,20 @@ void GlobalOptionsDialog::apply() {
 			ConfMan.set("gui_renderer", newGfxConfig, _domain);
 #ifdef USE_TRANSLATION
 			if (newCharset != TransMan.getCurrentCharset()) {
-				MessageDialog error(_("Theme does not support selected language"));
+				MessageDialog error(_("Theme does not support selected language!"));
 				error.runModal();
 			} else {
 #endif
-				MessageDialog error(_("Failed loading new settings"));
+				MessageDialog error(_("Theme cannot be loaded!"));
 				error.runModal();
 #ifdef USE_TRANSLATION
 			}
 			TransMan.setLanguage(oldLang);
 			_guiLanguagePopUp->setSelectedTag(selectedLang);
 			ConfMan.set("gui_language", oldLang);
 #endif
+			MessageDialog error(_("Misc settings will be restored."));
+			error.runModal();
```

Comment:
Rather that display two separate ErrorDialog, I think it would be better to concatenate the two messages and use a single dialog.